packages/core/src/Box/Box.stories.tsx

```diff
@@ -152,6 +152,26 @@ export const DirectionalMargin = () => (
   </Box>
 )
 
+export const Overflow = () => (
+  <Box p={3}>
+    <Box overflow='hidden' maxHeight='100px' maxWidth='100px' mb={3} color='black' bg='blue'>
```

Comment:
for new code could you use `color='background.darkest'` and `bg='primary'` if that works please